lib/ret/project.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Ret.Project do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Ret.{Repo, Project, OwnedFile}
+
+  @schema_prefix "ret0"
+  @primary_key {:project_id, :id, autogenerate: true}
+
+  schema "projects" do
+    field(:project_sid, :string)
+    field(:name, :string)
+    belongs_to(:created_by_account, Ret.Account, references: :account_id)
+    belongs_to(:project_owned_file, Ret.OwnedFile, references: :owned_file_id)
+    belongs_to(:thumbnail_owned_file, Ret.OwnedFile, references: :owned_file_id)
+
+    timestamps()
+  end
+
+  def project_by_sid(sid) do
+    Project |> Repo.get_by(project_sid: sid)
+  end
+
```

Comment:
the `to_sid` function was overloaded to deal with `Scene` and `SceneListing`s, so you probably can inline it in this case since I don't think there's an analog